Locales/frFR.lua

```diff
@@ -1,6 +1,133 @@
+--Reserved space below so all localization files line up
 
-if not (GetLocale() == "frFR") then return end;
```

Comment:
Keep this so it doesn't overwrite the locales on non-french clients.

Reply:
Oh, sorry, i'm not familiar with LUA, but i keep learning.